content/docs/studio/troubleshooting.md

```diff
@@ -329,20 +326,20 @@ This error indicates that the `dvc.lock` file in the given commit has an invalid
 YAML. If the given commit is unimportant to you, you can ignore this error.
 
 One potential cause for this error is that at the time of the given commit, your
-repository used DVC 1.0. The format of lock files used in DVC 1.0 were
-deprecated in the DVC 2.0 release. Upgrading to the latest DVC version will
-resolve this issue for any future commits in your repository.
+repository used DVC 1.0. The format of lock files used in DVC 1.0 was deprecated
+in the DVC 2.0 release. Upgrading to the latest DVC version will resolve this
+issue for any future commits in your repository.
 
 ## Project does not reflect updates in the Git repository
 
 When there are updates (new commits, branches, etc.) in your Git repository,
 your project in Iterative Studio gets reflected to include those updates. If the
 project has stopped receiving updates from the Git repository and you have to
-"force import" the project each time to get any new commit, then it is possible
+`force import` the project each time to get any new commit, then it is possible
 that the Iterative Studio webhook in your repository got deleted or messed up.
 
 Iterative Studio periodically checks for any missing or messed up webhooks, and
-attempts to re-create it. Currently, this happens every 2 hours. The webhook
+attempts to re-create them. Currently, this happens every 2 hours. The webhook
```

Comment:
@amritghimire Are there any any steps that the user can manually take to resolve this? If yes, can you pls describe them?

Reply:
Force import should take care of this.

Reply:
@amritghimire Does this mean that if the webhook is messed up and the user has to run force import to update the project table, then from the next time onwards the project table will get updated automatically? (coz the webhook got recreated when the user ran force import for the first time)?